jwst/datamodels/util.py

```diff
@@ -6,6 +6,11 @@
 import warnings
 import os
 from os.path import basename
+from platform import system as platform_system
+from plistlib import loads as plist_loads
+import psutil
+from psutil._common import bytes2human
```

Comment:
This is pretty short, self-contained function:

https://psutil.readthedocs.io/en/latest/#bytes-conversion

Maybe worth vendorizing so we don't have to worry about that underscore module changing?